noir/noir-repo/noir_stdlib/src/embedded_curve_ops.nr

```diff
@@ -42,7 +44,7 @@ impl Neg for EmbeddedCurvePoint {
 #[foreign(multi_scalar_mul)]
 // docs:start:multi_scalar_mul
 pub fn multi_scalar_mul<N>(
-    points: [Field; N], // points represented as x and y coordinates [x1, y1, x2, y2, ...]
+    points: [Field; N], // points represented as x and y coordinates [x1, y1, x2, y2,...]
     scalars: [Field; N] // scalars represented as low and high limbs [low1, high1, low2, high2, ...]
```

Comment:
huh... this should be `2*N` and `N`, right? We've got half as many points as scalars here.

Reply:
If we switch to EmbeddedCruvePoints, then yes il will be N points and 2*N scalars.

Reply:
Ah, I forgot that the scalars are split into high and low limbs so these currently line up.